gtfs-realtime/proto/gtfs-realtime.proto

```diff
@@ -717,3 +729,280 @@ message TranslatedString {
+}
+
+// Defines the geographical representation of a point in a shape.
+message ShapePoint {
+  // Latitude of a shape point. Each record in shapes.txt represents a shape point used to define the shape.
+  required float shape_pt_lat = 1;
+  // Longitude of a shape point.
+  required float shape_pt_lon = 2;
+  // Actual distance traveled along the shape from the first shape point to the point specified in this record.
+  optional float shape_dist_traveled = 3;
+  // Sequence in which the shape points connect to form the shape. Values must increase along the trip but do not need to be consecutive.
+  required uint32 shape_pt_sequence = 4;
+}
+
+// Describes new routes. A route is a group of trips that are displayed to riders as a single service.
+message Route {
+  //  Identifier of the route. Must be different than any route_id defined in the GTFS.
+  required string route_id = 1;
+  // Identifies a transit brand which is often synonymous with a transit agency.
+  required string agency_id = 2;
+  // Short name of a route.
+  optional TranslatedString route_short_name = 3;
+  // Full name of a route. This name is generally more descriptive than the route_short_name and often includes the route's destination or stop.
+  optional TranslatedString route_long_name = 4;
+  // Description of a route that provides useful, quality information. Do not simply duplicate the name of the route.
+  optional TranslatedString route_desc = 5;
+  // Indicates the type of transportation used on a route.
+  required RouteType route_type = 6;
+  // URL of a web page about the particular route. Should be different from the agency.agency_url value.
+  optional TranslatedString route_url = 7;
+  // Route color designation that matches public facing material. D
+  optional string route_color = 8;
+  // Legible color to use for text drawn against a background of route_color.
+  optional string route_text_color = 9;
+  // Orders the routes in a way which is ideal for presentation to customers. Routes with smaller route_sort_order values should be displayed first.
+  optional uint32 route_sort_order = 10;
+
+  // Valid options for route_type:
+  // 0 - Tram, Streetcar, Light rail. Any light rail or street level system within a metropolitan area.
+  // 1 - Subway, Metro. Any underground rail system within a metropolitan area.
+  // 2 - Rail. Used for intercity or long-distance travel.
+  // 3 - Bus. Used for short- and long-distance bus routes.
+  // 4 - Ferry. Used for short- and long-distance boat service.
+  // 5 - Cable car. Used for street-level cable cars where the cable runs beneath the car.
+  // 6 - Gondola, Suspended cable car. Typically used for aerial cable cars where the car is suspended from the cable.
+  // 7 - Funicular. Any rail system designed for steep inclines.
+  enum RouteType {
+    LIGHT_RAIL = 0;
+    SUBWAY = 1;
+    RAIL = 2;
+    BUS = 3;
+    FERRY = 4;
+    CABLE_CAR = 5;
+    GONDOLA = 6;
+    FUNICULAR = 7;
+  }
```

Comment:
Here new route types are missing: TROLLEY_BUS (11), MONORAIL(12)

Reply:
@lionel-nj Good catch! Could you go ahead and add them? Let's keep the numbering consistent - TROLLEY_BUS = 11, MONORAIL = 12.

Reply:
My bad, 4c2a43aa65dd7e25efffd9b60123a6444cec99af adds these descriptions.